app/views/devise/mailer/unlock_instructions.html.erb

```diff
@@ -0,0 +1,7 @@
+<p>Hello <%= @resource.email %>!</p>
```

Comment:
railsdevs doesn't use locking, so we can delete this file along with the text version.